Creiwry/irl-meeting-app
# Feat back registrations add fields

### Ticket Link
(Found on Jira in your ticket description)
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-18

### User Story
(Linked to on Jira in your ticket description)
https://templatethpprojetfinal.atlassian.net/browse/THPPROJ-18

### What does this PR do?
(Your answer here.)
Adds 3 custom fields as dropdown menu options (gender, sexual_orientation, relationship_status) to be added an saved to user profile on the registrations#edit page

### Where should the reviewer start?
(Your answer here.)
/users/edit

### How should this be manually tested?
(Your answer here.)
Try signing in, modifying your gender, relationship status, orientation, then go to the view of your page (top of edit page) to ensure that the changes are displayed

### Screenshots (if appropriate)
(Your answer here.)
![Screen Shot 2023-09-13 at 23 15 28](https://github.com/Creiwry/irl-meeting-app/assets/137942686/25151d61-ff6a-48d0-8560-5351d5c3d9a6)
![Screen Shot 2023-09-13 at 23 16 02](https://github.com/Creiwry/irl-meeting-app/assets/137942686/94d9c88a-62eb-46e4-85a2-2a36e2be18e7)

### Are there any breaking changes?
No. rspec tests good
